bindings/python/tools/install_boost.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+
+set -e
+set -x
+
+# Don't re-build and re-install Boost if it was already installed previously
+if [[ "$OSTYPE" == "msys" ]] && [[ -f /c/Boost/bin/b2.exe ]]
+then
+  echo Using cache...
+  exit 0
+elif [[ "$OSTYPE" == "darwin"* ]] && [[ -f /usr/local/bin/b2 ]]
+then
+  echo Using cache...
+  exit 0
+fi
+
+# Download Boost sources
+curl -L https://dl.bintray.com/boostorg/release/1.73.0/source/boost_1_73_0.tar.gz -o /tmp/boost.tar.gz
+tar xzf /tmp/boost.tar.gz -C /tmp
+
+# Install Boost from Homebrew on macOS
+if [[ "$OSTYPE" == "darwin"* ]]
+then
+  brew update
+  brew install boost boost-build
```

Comment:
if you don't use the downloaded boost archive (from bintray), you should probably do this before downloading and unzipping it

Reply:
I use downloaded archive in other script so I can build Boost.Python for specific Python version. But I also download archive in that script (if it is not already downloaded) so it shouldn't really matter.